.github/workflows/skip-k3d.yml

```diff
@@ -0,0 +1,17 @@
+name: e2e-k3d
+on:
+  pull_request:
+    paths:
```

Comment:
I think the 'skip' tests need to use `paths-ignore` rather than `paths`.

The way I understand it, if someone commits a PR with docs AND code changes, the `skip-k3d` workflow will run because the docs changes trigger the `paths` conditional. This will pass the workflows and we could end up merging bad code because the real `e2e-k3d` test will still be running.

Reply:
And the real tests should be using `paths` instead of `paths-ignore`